engine/src/test/scala/engine/backend/jes/JesBackendSpec.scala

```diff
@@ -64,12 +64,17 @@ class JesBackendSpec extends FlatSpec with Matchers with Mockito with BeforeAndA
     call.task returns task
     val backendCallKey = mock[BackendCallKey]
     backendCallKey.attempt returns 1
+    backendCallKey.scope returns call
     val backendCall = mock[JesBackendCall]
     backendCall.call returns call
     backendCall.key returns backendCallKey
     backendCall.preemptible returns false
     backendCall.maxPreemption returns 1
     backendCall.workflowDescriptor returns wd
+    val jobDescriptor = mock[BackendCallJobDescriptor]
```

Comment:
Some methods inside jobDescriptor are calling backend methods (for runtime attributes etc..) which are really annoying to get to return what you want for the purpose of the test without mocking, but I'll see what I can do.